Ffyrn.MonoEngine/Class1.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Ffyrn.MonoEngine
+{
+    public class Class1
```

Comment:
Rename this to `Game` or something similar, add something basic, and implement that into the example game please !!! 
What is on the main branch has to be runnable at all times, and at the moment there is no code to run lol

Reply:
By "something simple" I do mean something simple, don't go implementing a feature. Display "Hello world" on a monogame window or something!

Reply:
Added a simple `DrawRectangle` method in b03663e.

I also tried to add unit tests but it seems we can't mock `GraphicsDevice` easily, I have left it in commented out as an example though.